bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -21,3 +21,7 @@ extendedArguments:
   service-cluster-ip-range: {{range .ServiceClusterIPRange}}
   - {{.}}{{end}}
   {{end}}
+  pv-recycler-pod-template-filepath-nfs:
+  - ""
+  pv-recycler-pod-template-filepath-hostpath:
+  - ""
```

Comment:
The bootstrap KCM won't find the template file.

@smarterclayton should I override the config here or should I create the file (in the installer?)?